journal/updated/01-journal.adoc

```diff
@@ -153,82 +159,84 @@ or people of lower social or economic position.
 rather than showing preferment by addressing certain individuals in the plural.]
-for I did acknowledge to him,
-that it was the language of God to Adam, and the language of the scripture; but, said I,
-that is not enough for us now in this day, we must have a command for it.
-To which he answered, "`Hold fast the form of sound words, which you have heard of me.`"
-I asked him, Whether that was scripture; he asked me, whether I would deny it;
-I told him, he was to prove it.
-Then he took the bible out of my hand, and he turned to 2 Tim. 1:13, which he read,
-and told me, that Hold fast there, was a command; which I knew very well,
-both the scripture and the command: but to prove him further,
-I desired him to read a little more of that chapter, both backward and forward,
-which he freely did, and asked me, Why I did require that of him?
-I told him, that we heard the Quakers denied the scripture,
+for I acknowledged
+that it was the language of God to Adam, and the language of the scripture,
+"`but,`" said I,
+"`that is not enough for us now in this day, we must have a command for it.`"
+To which he answered, "`Hold fast the form of sound words,
+which you have heard of me.`" I asked him whether that was scripture;
+he asked me whether I would deny it; I told him he must to prove it.
+Then he took the bible out of my hand, and turned to 2 Tim. 1:13, which he read,
+and told me that "`Hold fast`" there was a command, which I knew very well,
+both the scripture and the command. But to prove him further,
+I desired him to read a little more of that chapter, both before and after,
+which he freely did, and asked me why I asked him to do so.
+I told him that we heard the Quakers denied the scripture,
 and that they would not read them.
 He said there were many false reports of them.
-And truly when he read the scripture so readily, I concluded in myself,
+And truly when he read the scripture so readily, I concluded in myself
 that what was reported of them was not true;
 and he saw that he had reached to the witness of God in me.
 Then he exhorted me to take heed to that light that shined in my heart,
-and did show me my vain thoughts,
-and reprove me in secret for every idle word and action; saying
+which showed me my vain thoughts,
+and reproved me in secret for every idle word and action, saying
 that "`that was the true light,
-which enlightens every man that comes into the world;`" and in that light,
-I should see more light, and that would open the scriptures to me,
-and that I should receive a measure of the same spirit that gave them forth.
-Further he told me, It was "`the more sure word of prophecy,
+which enlightens every man that comes into the world.`" And in that light,
+I would see more light, which would open the scriptures to me,
+so that I would receive a measure of the same Spirit that gave them forth.
+He further told me it was "`the more sure word of prophecy,
```

Comment:
The "It" here refers to the Spirit or Light. So I think I would clarify this part, saying:  He further told me, that this Spirit or Light was "the more sure word of prophecy...."